src/main/java/VASSAL/build/module/map/KeyBufferer.java

```diff
@@ -66,6 +68,11 @@
   protected Point anchor;
   protected Color color = Color.black;
   protected int thickness = 3;
+  protected GamePiece bandSelectPiece = null;
+  
+  private static final int NOT_BAND_SELECT     = 0;
```

Comment:
Safer to use enums for this, else someone some day will get the idea to assign `bandSelect = 27`

And it's easy, define the enum at the bottom of this class:
```
  private enum BandSelectType {
    NOT,
    NORMAL,
    SPECIAL;
  }
```

Then use it just like you would use the int:
```
BandSelectType bandSelect = BandSelectType.NOT;
(...)
bandSelect = BandSelectType.SPECIAL;
(...)
if (bandSelect != BandSelectType.NOT) {
```

Reply:
added, thanks